contracts/VeERC20.sol

```diff
@@ -0,0 +1,150 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.6.12;
```

Comment:
Can we use a more recent version of solidity for new contracts? This is possible with hardhat see https://hardhat.org/guides/compile-contracts.html#multiple-solidity-versions

Reply:
ooo wow didn't know you could do this! I updated this to use `0.8.6` but one issue I ran into is that the existing `openzeppelin` contracts in the repo are using an outdated version and I can't upgrade it b/c it would break the existing contracts so I instead manually copied over the relevant latest openzeppelin contracts directly into the repo. i think this is fine but curious what you think